homeassistant/components/flux_led/__init__.py

```diff
@@ -19,14 +37,75 @@ async def async_setup(hass: HomeAssistant, config: dict):
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Set up Flux LED/MagicLight from a config entry."""
 
-    hass.async_create_task(
-        hass.config_entries.async_forward_entry_setup(entry, "light")
-    )
+    conf = entry.data
+    options = entry.options
+
+    hass.data[DOMAIN][entry.entry_id] = {}
+
+    if options.get("global", {}).get(CONF_AUTOMATIC_ADD, conf[CONF_AUTOMATIC_ADD]):
+        bulb_list = FluxLedList(
+            hass,
+            devices=conf[CONF_DEVICES],
+            config_entry=entry,
+        )
+
+        hass.data[DOMAIN][entry.entry_id] = {BULB_COORDINATOR: bulb_list}
```

Comment:
Please call the bulb discovery instance constant something else than coordinator.

Reply:
OK updated.

Reply:
Removed entirely